docker/docker.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+
+readonly ARREBOL_REPO=ufcglsd/arrebol
+readonly ARREBOL_CONTAINER=arrebol
+
+build() {
+  local DOCKERFILE_DIR=docker/Dockerfile
+  local TAG="${1-latest}"
+  mvn clean install -DskipTests
+  docker build --tag "${ARREBOL_REPO}":"${TAG}" \
+          --file "${DOCKERFILE_DIR}" .
+}
+
+run() {
+  local TAG="${1-latest}"
+  local PORT="${2-8080}"
+  docker run -dit \
+    --name "${ARREBOL_CONTAINER}" \
+    -p ${PORT}:8080 \
+    -v "$(pwd)"/src/main/java/resources/arrebol.json:/service/config/arrebol.json \
+    -v "$(pwd)"/src/main/java/resources/application.properties:/service/config/application.properties \
+    "${ARREBOL_REPO}":"${TAG}"
+}
+
+publish() {
+  local tag="${1:-latest}"
```

Comment:
${1:-latest} or ${1-:latest} ?

Reply:
or even ${1-latest}

Reply:
This means to use the `latest` value by default if the `$1` variable is empty.

Reply:
isnt the ':' a typo?

Reply:
It is not.

Reply:
@wesleymonte add a comment explaining this ":". it is not trivial

Reply:
Done.